lambda_function.py

```
@@ -123,6 +128,55 @@ def convert_coordinates(x, y, z, ref_lat, ref_lon, ref_ele, rotation_angle=0, sc
         print(f"Error in coordinate conversion: {e}")
         return ref_lon, ref_lat, ref_ele
 
+def quaternion_to_euler(qx, qy, qz, qw):
```

Ruff (0.8.2)

Missing return type annotation for public function `quaternion_to_euler`

(ANN201)